internal/client.go

```diff
@@ -441,6 +446,15 @@ type (
 		CheckCallAllowed(ctx context.Context, method string, req, reply interface{}) error
 	}
 
+	// UpdateWorkflowExecutionOptions encapsulates the optional parameters for
+	// sending an update to a workflow execution.
+	// NOTE: Experimental
+	UpdateWorkflowExecutionOptions struct {
+		FirstExecutionRunID string
```

Comment:
Where is update ID provided? Are you adding this later when we switch to the new protocol?

Reply:
Yes, this PR is interaction based which was from the RequestID design. Next PR to API brings in client-specified UpdateID